doc/developer/animation.md

```diff
@@ -21,8 +21,8 @@ There are two ways data can be animated using `Ra::Core::Animation::KeyFramedVal
     struct MyStruct {
```

Comment:
Theses snippets could be moved as a unit tests and referenced here.

Reply:
This is out of the scope of this PR, but sure that would be great.

Reply:
Well, I'll do it, because the current code in the doc do not compile (mistyped param name, struct could is a reserved keyword, .hpp missing in headers ... I have started but still some work to do)

Reply:
done.